service/builder/exporters_builder.go

```diff
@@ -53,6 +73,19 @@ func (exp *builtExporter) Shutdown() error {
 // Exporters is a map of exporters created from exporter configs.
 type Exporters map[configmodels.Exporter]*builtExporter
 
+// StartAll starts all exporters.
+func (exps Exporters) StartAll(logger *zap.Logger, host exporter.Host) error {
+	for cfg, exp := range exps {
+		logger.Info("Exporter is starting...", zap.String("exporter", cfg.Name()))
+
+		if err := exp.Start(host); err != nil {
+			return err
+		}
+		logger.Info("Exporter is started.", zap.String("exporter", cfg.Name()))
```

Comment:
nit: exporter started. (no need to have is)

Reply:
Fixed.